src/http/http-service.ts

```diff
@@ -118,7 +118,7 @@ export class HttpService {
   ) {
     const { policy, gotOptions } = this.parseOptions(options);
 
-    this.logger.debug({ method: 'getJson', url });
+    this.logger.debug({ method: 'getJson', url, gotOptions });
```

Comment:
I didn't include `gotOptions` because It might include sensitive data, the same could be argued about the payload

Reply:
tbh we leak api keys and other sensitive data all over. Was thinking about enabling debug logs in dev only. And possibly write a util to obfuscate sensitive data with a given js object